radio/src/targets/taranis/hal.h

```diff
@@ -1962,7 +1964,7 @@
 // TELEMETRY_EXTI IRQ
 #if !defined(USE_EXTI9_5_IRQ)
   #define USE_EXTI9_5_IRQ
-  #define EXTI9_5_IRQ_Priority 5
+  #define EXTI9_5_IRQ_Priority          TELEMETRY_EXTI_PRIO
```

Comment:
Sorry for the mess, but I think we could have something like:
```c
#if !defined(USE_EXTI9_5_IRQ)
  #define USE_EXTI9_5_IRQ
#endif
// overwrite priority
#undef EXTI9_5_IRQ_Priority
#define EXTI9_5_IRQ_Priority          TELEMETRY_EXTI_PRIO
```

That should allow for removing the other definitions/changes in this file.

Reply:
done